tests/www/test_views.py

```diff
@@ -1854,6 +1856,24 @@ def test_extra_links_no_response(self, get_dag_function):
             'url': None,
             'error': 'No URL found for no_response'})
 
+    @mock.patch('airflow.www.views.dagbag.get_dag')
+    def test_operator_extra_link_override_plugin(self, get_dag_function):
+        get_dag_function.return_value = self.dag
+
+        response = self.client.get(
+            "{0}?dag_id={1}&task_id={2}&execution_date={3}&link_name=airflow".format(
+                self.ENDPOINT, self.dag.dag_id, self.task_2.task_id, self.DEFAULT_DATE),
+            follow_redirects=True)
+
+        self.assertEqual(response.status_code, 200)
+        response_str = response.data
+        if isinstance(response.data, bytes):
+            response_str = response_str.decode()
+        self.assertEqual(json.loads(response_str), {
+            'url': 'https://airflow.apache.org/1.10.5/',
```

Comment:
I'm not quite sure what this is testing, nor where this string is coming from in the code?

Reply:
It is coming from here:
https://github.com/apache/airflow/blob/d93bb34ca08d45d4dce77bc0e6415b4251c53b3d/tests/plugins/test_plugin.py#L113-L118

It is testing that if OperatorLink is added in the plugin, the same link appears

Reply:
Let me move them to separate file

Reply:
A comment explaining what is going on here would help I think (that the link is being overridden by one registered via a plugin)

Reply:
Added the comment